panic_handlers/small_panic/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![no_std]
+
+use libtock_low_level_debug::{AlertCode, LowLevelDebug};
+use libtock_platform::Syscalls;
+use libtock_runtime::TockSyscalls;
+
+#[panic_handler]
+fn panic_handler(_info: &core::panic::PanicInfo) -> ! {
+    // Signal a panic using the LowLevelDebug capsule (if available).
+    LowLevelDebug::<TockSyscalls>::print_alert_code(AlertCode::Panic);
+
+    // Exit with a non-zero exit code to indicate failure.
+    TockSyscalls::exit_terminate(1);
```

Comment:
I believe this is the first place we exit with a nonzero completion code. The meanings of the completion code have not been defined yet, so making this 1 will have an impact on future compatibility of exit codes.

Would you mind sending a TRD PR into upstream Tock to start the discussion on completion code definitions?

Reply:
Making it 0 would _also_ have an impact on future compatibility of exist codes for the exact same reason, so there's no way I could put any exit code here without starting this discussion.

I'll send something small soon, along the lines of "0 is success, non-zero should be non-success but is implementation defined, with possible extension in the future"

Reply:
https://github.com/tock/tock/pull/2914

Reply:
Using your proposed completion code design, I think this should be rewritten as:

```
use libtock_platform::ErrorCode;
TockSyscalls::exit_terminate(ErrorCode::Fail as u32);
```

Reply:
Good idea, done.